pkg/network/state.go

```diff
@@ -204,7 +202,7 @@ func (ns *networkState) addDNSStats(id string, conns []ConnectionStats) {
 
 		if dnsStats, ok := ns.clients[id].dnsStats[key]; ok {
 			conn.DNSTimeouts = dnsStats.timeouts
-			conn.DNSSuccessfulResponses = dnsStats.countByRcode[uint8(layers.DNSResponseCodeNoErr)]
+			conn.DNSSuccessfulResponses = dnsStats.countByRcode[0] // 0 means No Error
```

Comment:
Can you expand the comment to describe why you aren't including layers?

Reply:
I think it would be preferable to still define `0` as a constant, but here in this package.

Reply:
Defining it as a constant would be good, too, but I want to answer the future question of "why don't they just import layers?"

Reply:
@brycekahle , @heyronhay - updated the PR based on your comments. Please have a look when you get chance.